### PR TITLE
FSA-21 Create endpoint for fetch list of all movies in database

### PR DESCRIPTION
**Pull request by DZietara**

### CODE REVIEW

File: src/test/java/pl/aogiri/hhu/fsa/backend/movie/application/mapper/ScoreEntityFixture.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    public static ScoreEntity johnDon_theIncredibles_score(String score) {
+        final var scoreEntity = new ScoreEntity();
+        scoreEntity.setUser(UserEntityFixture.johnDoe());
+        scoreEntity.setScore(new BigDecimal(score));
+        return scoreEntity;
+    }
+
+    public static ScoreEntity janeDoe_theIncredibles_score(String score) {
+        final var scoreEntity = new ScoreEntity();
+        scoreEntity.setUser(UserEntityFixture.janeDoe());
```
Comment (KanekiSenapi) [inline src/test/java/pl/aogiri/hhu/fsa/backend/movie/application/mapper/ScoreEntityFixture.java:18]: setaMovie

File: src/test/java/pl/aogiri/hhu/fsa/backend/movie/application/mapper/ScoreEntityFixture.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+import pl.aogiri.hhu.fsa.backend.movie.domain.entity.ScoreEntity;
+
+import java.math.BigDecimal;
+
+public class ScoreEntityFixture {
+
+    public static ScoreEntity johnDon_theIncredibles_score(String score) {
+        final var scoreEntity = new ScoreEntity();
+        scoreEntity.setUser(UserEntityFixture.johnDoe());
```
Comment (KanekiSenapi) [inline src/test/java/pl/aogiri/hhu/fsa/backend/movie/application/mapper/ScoreEntityFixture.java:11]: setMovie

File: src/test/java/pl/aogiri/hhu/fsa/backend/movie/application/MovieServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    private MovieRepository movieRepository;
+
+    @InjectMocks
+    private MovieService movieService;
+
+    @Test
+    void shouldReturnAllMovies() {
+        //given
+        final var movies = List.of(MovieEntityFixture.theIncredibles());
+
```
Comment (KanekiSenapi) [inline src/test/java/pl/aogiri/hhu/fsa/backend/movie/application/MovieServiceTest.java:32]: space delete

File: src/test/java/pl/aogiri/hhu/fsa/backend/movie/application/mapper/MovieEntityFixture.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+    public static MovieEntity theIncredibles() {
+        final var movieEntity = new MovieEntity();
+        movieEntity.setId(1L);
+        movieEntity.setTitle("The Incredibles");
+        movieEntity.setDescription("The Incredibles is a 2004 American computer-animated superhero film written and directed by Brad Bird. The film is set in an alternate version of the 1960s, where superheroes are forced to live in secrecy due to public backlash. The story follows Bob Parr (Mr. Incredible), a superhero who is forced to live a normal life as a suburban insurance claims adjuster. His wife, Helen (Elastigirl), and their three children also have superpowers, but they must keep them hidden to avoid being discovered.");
+        movieEntity.setScores(List.of(
+                ScoreEntityFixture.johnDon_theIncredibles_score("8"),
+                ScoreEntityFixture.janeDoe_theIncredibles_score("9"),
+                ScoreEntityFixture.johnDon_theIncredibles_score("8")
```
Comment (KanekiSenapi) [inline src/test/java/pl/aogiri/hhu/fsa/backend/movie/application/mapper/MovieEntityFixture.java:18]: 2x same score

File: src/test/java/pl/aogiri/hhu/fsa/backend/movie/application/mapper/ScoreEntityFixture.java (hunk 1 above)
Comment (DZietara) [reply src/test/java/pl/aogiri/hhu/fsa/backend/movie/application/mapper/ScoreEntityFixture.java:18]: corrected

File: src/test/java/pl/aogiri/hhu/fsa/backend/movie/application/mapper/ScoreEntityFixture.java (hunk 2 above)
Comment (DZietara) [reply src/test/java/pl/aogiri/hhu/fsa/backend/movie/application/mapper/ScoreEntityFixture.java:11]: corrected

File: src/test/java/pl/aogiri/hhu/fsa/backend/movie/application/MovieServiceTest.java (hunk 3 above)
Comment (DZietara) [reply src/test/java/pl/aogiri/hhu/fsa/backend/movie/application/MovieServiceTest.java:32]: corrected

File: src/test/java/pl/aogiri/hhu/fsa/backend/movie/application/mapper/MovieEntityFixture.java (hunk 4 above)
Comment (DZietara) [reply src/test/java/pl/aogiri/hhu/fsa/backend/movie/application/mapper/MovieEntityFixture.java]: corrected

**Review by KanekiSenapi — APPROVED** [review]

**Pull request merged**